src/field/captcha/third_party_captcha.jsx

```diff
@@ -31,9 +33,25 @@ const getCaptchaProvider = provider => {
       return window.friendlyChallenge;
     case ARKOSE_PROVIDER:
       return window.arkose;
+    case AUTH0_V2_CAPTCHA_PROVIDER:
+      return window.turnstile;
   }
 };
 
+export const getRenderParams = (self) => {
```

Comment:
Can we move this as part of the React component? We shouldn't be passing around `this` outside the component when we can have it as a class method for the component.

Reply:
Code coverage made me do it! I can pass the individual params instead of this but there was no good way (that I could find) to get coverage without pulling it out. Any suggestions?

Reply:
I was looking this up and found [this](https://stackoverflow.com/questions/54942892/jest-enzyme-test-a-function-call-in-componentdidmount). I guess one thing you could do is if there's a mock window object, then you can follow that to do what the stackoverflow answer does and that would call componentDidMount -> injectCaptchaScript -> attaches callback function that calls getRenderParams to window -> call callback from window function to unit test. But I like your current approach better. Only thing I would change is instead of individually passing the render params, can we pass an object that contains the render params so it's extensible for the future.

Reply:
Let me know if this implementation works for you!

Reply:
I added a review comment around this specifically. I do not want to be blocking, but I believe this can be improved.

Reply:
> because most of the code here is untested anyway

I think this shouldnt mean we can not try our best to improve the quality and add tests for new added functionality. Leaving that to you, as you own the functionality, but I would recommend ensuring things are tested.